src/SeleniumLibrary/keywords/javascript.py

```diff
@@ -73,19 +91,52 @@ def execute_async_javascript(self, *code):
         | ...         | window.setTimeout(answer, 2000);                |
         | `Should Be Equal` | ${result} | text |
         """
-        js = self._get_javascript_to_execute(code)
-        self.info("Executing Asynchronous JavaScript:\n%s" % js)
-        return self.driver.execute_async_script(js)
-
-    def _get_javascript_to_execute(self, lines):
-        code = ''.join(lines)
-        path = code.replace('/', os.sep)
-        if os.path.isabs(path) and os.path.isfile(path):
-            code = self._read_javascript_from_file(path)
-        return code
+        js_code, js_args = self._get_javascript_to_execute(code)
+        self.info('Executing Asynchronous JavaScript:\n%s\nBy using argument(s):\n"%s"'
+                  % (js_code, ', '.join(js_args)))
```

Comment:
Same comment about logging as above. Logging should probably be done using a helper.

Reply:
Helper is good idea. Will definitely take a look.